Library/ViewModels/PledgeViewModel.swift

```diff
@@ -31,43 +40,90 @@ public protocol PledgeViewModelType {
 public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, PledgeViewModelOutputs {
   public init() {
     let projectAndReward = Signal.combineLatest(
-      self.configureProjectAndRewardProperty.signal, self.viewDidLoadProperty.signal
+      self.configureProjectAndRewardProperty.signal,
+      self.viewDidLoadProperty.signal
     )
     .map(first)
     .skipNil()
 
     let project = projectAndReward.map(first)
     let reward = projectAndReward.map(second)
-    let isLoggedIn = projectAndReward
-      .map { _ in AppEnvironment.current.currentUser }
-      .map(isNotNil)
+    let isLoggedIn = projectAndReward.map { _ in AppEnvironment.current.currentUser }.map(isNotNil)
+
+    let pledgeAmount = Signal.merge(
+      self.pledgeAmountSignal,
+      projectAndReward.map { $1.minimum }
+    )
+
+    let shippingRulesEvent = projectAndReward
+      .filter { _, reward in reward.shipping.enabled }
+      .switchMap { (project, reward) -> SignalProducer<Signal<[ShippingRule], ErrorEnvelope>.Event, Never> in
+        AppEnvironment.current.apiService.fetchRewardShippingRules(projectId: project.id, rewardId: reward.id)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .map(ShippingRulesEnvelope.lens.shippingRules.view)
+          .retry(upTo: 3)
+          .prefix(value: [])
+          .materialize()
+      }
+
+    self.presentShippingRules = shippingRulesEvent.values()
```

Comment:
Is this just a placeholder for now? Perhaps add a comment and set to `.empty`?

Reply:
Yup, it's a placeholder...I'm using it right in the follow up PR so would be undoing some unnecessary work (plus altering tests)...let's keep it this way for now?

Reply:
Sounds good